corehq/toggles.py

```diff
@@ -1808,3 +1808,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_CUSTOM,
     [NAMESPACE_DOMAIN]
 )
+
+
+LOCATION_RATIONALIZATION = StaticToggle(
+    'location_rationalization',
+    'Ability to reorder organization structure for location rationalization',
+    TAG_CUSTOM,
+    [NAMESPACE_USER]
```

Comment:
User-based flags always catch my eye, why does this feature need one?

Reply:
We wont want to give ability to all users to upload requests to this.
Is there a different way it should be done?

Reply:
I don't like blurring the line between permissions and feature flags. On the other hand, adding a permission is more work than a feature flag without much benefit, so, yeah, let's leave it as-is.